gradle.properties

```diff
@@ -1,8 +1,8 @@
 org.gradle.caching=true
 org.gradle.jvmargs='-Dfile.encoding=UTF-8'
 group=org.ballerinax.azurefunctions
-version=1.2.19-SNAPSHOT
-ballerinaLangVersion=1.2.18
+version=1.2.19
```

Comment:
This should be `1.2.20-SNAPSHOT` right? Seems like the Gradle plugin commit is overridden.

Reply:
Fixed it.